tests/foreman/ui/test_settings.py

```diff
@@ -175,6 +175,60 @@ def test_negative_validate_error_message(session, set_original_property_value):
                 assert is_valid_error_message(str(context.value))
 
 
+@tier2
+def test_positive_selectors(session):
+    """"Testing input for selectors: dropdown, text area, input box
+
+    :id: 529ddd3a-1271-4043-9006-eac436b08b11
+
+    :expectedresults: Successfully add value to different selectors
+
+    :CaseImportance: High
+    """
+    uuid_input = f'[ {gen_string("alpha")} ]'
+    random_input = gen_string('alpha')
+    uuid_duplicate_prop = 'host_dmi_uuid_duplicates'
+    reg_host_prop = 'register_hostname_fact'
+    dep_solve_prop = 'content_view_solve_dependencies'
+    with session:
+        # Drop down selector
+        session.settings.update(
+            'name = {}'.format(dep_solve_prop),
+            'Yes'
+        )
+        result = session.settings.read('name = {}'.format(dep_solve_prop))
+        assert result['table'][0]['Value'] == 'Yes'
+        # reset input value
+        session.settings.update(
+            'name = {}'.format(dep_solve_prop),
+            'No'
+        )
```

Comment:
Nitpick: technically, someone/something else could have set this value before this test runs.
Just to be on safe side, we should go: read current value and store in variable, change, read new value and verify it's set correctly, change to old value (from variable).
Can Nailgun read settings? So we get initial value using API, and only update/re-read part with Airgun.
Do you think this is something you could try? Or rather deal with this if it ever becomes a problem?

Reply:
I can try and see.  I'll try both yours and @omkarkhatavkar's suggestion.

Reply:
@mirekdlugosz, I've implemented what you suggested and it works well.  Below is the test result.  I've yet to try with API call.

```
]$ pytest tests/foreman/ui/test_settings.py::test_positive_selectors
===================== test session starts =================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/ltran/Projects/robottelo
plugins: services-1.3.1, mock-1.10.4
collecting ... 2019-11-26 22:15:10 - conftest - DEBUG - Collected 1 test cases
2019-11-26 17:15:11 - robottelo.ssh - DEBUG - Instantiated Paramiko client 0x7f4ee4bbee80
2019-11-26 17:15:11 - robottelo.ssh - INFO - Connected to [dell-per6415-03.lab2.eng.bos.redhat.com]
2019-11-26 17:15:11 - robottelo.ssh - INFO - >>> rpm -q satellite
2019-11-26 17:15:12 - robottelo.ssh - INFO - <<< stdout
satellite-6.7.0-4.beta.el7sat.noarch

2019-11-26 17:15:12 - robottelo.ssh - DEBUG - Destroyed Paramiko client 0x7f4ee4bbee80
2019-11-26 17:15:12 - robottelo.host_info - DEBUG - Host Satellite version: 6.7
2019-11-26 17:15:12 - robottelo.helpers - DEBUG - Generated file bz_cache.json with BZ collect data
collected 1 item                                                                                                                                                                                                                                                           

tests/foreman/ui/test_settings.py .                                                                                                                                                                                                                                  [100%]

======================= warnings summary ===============
tests/foreman/ui/test_settings.py::test_positive_selectors
tests/foreman/ui/test_settings.py::test_positive_selectors
  /home/ltran/Projects/myvenv/lib64/python3.6/site-packages/widgetastic/widget/select.py:133: DeprecationWarning: The unescape method is deprecated and will be removed in 3.5, use html.unescape() instead.
    for option

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====== 1 passed, 2 warnings in 56.72 seconds ================
```